docs/technical_reference/property_models/ADM1.rst

```diff
@@ -0,0 +1,205 @@
+ADM1 Property Package
+=====================
+
+This package implements properties and reactions of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in
+`Batstone, D. J. et al. (2002) <https://iwaponline.com/wst/article-abstract/45/10/65/6034>`_ and `Rosen and Jeppsson (2006) <https://www.iea.lth.se/WWTmodels_download/TR_ADM1.pdf>`_.
+
+This Anaerobic Digestion Model no.1 (ADM1) property/reaction package:
+   * supports 'H2O', 'S_su', 'S_aa', 'X_fa', 'X_va', 'X_bu', 'X_pro', 'X_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', and 'S_co2' as components
+   * supports only liquid and vapor phase
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'X_fa', 'X_va', 'X_bu', 'X_pro', 'X_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', 'S_co2']"
+  "Phases", ":math:`p`", "['Liq', 'Vap']"
+
+Components
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable"
+
+  "Monosaccharides, S_su", ":math:`S_su`", "S_su"
```

Comment:
```suggestion
  "Monosaccharides, S_su", ":math:`S_{su}`", "S_su"
```
I think you need to wrap all the subscripts with more than one letter in curly braces as shown in this suggestion.